gcp_pilot/identity_platform.py

```diff
@@ -44,37 +48,116 @@ def create(cls, data: ResourceType) -> "User":
             email=data["email"],
             verified=data.get("emailVerified"),
             disabled=data.get("disabled"),
+            photo_url=data.get("photoUrl"),
             created_at=parse_timestamp(timestamp=data["createdAt"]),
             last_login_at=parse_timestamp(timestamp=data.get("lastLoginAt")),
             password_changed_at=parse_timestamp(timestamp=data.get("passwordUpdatedAt")),
             extra_attributes=json.loads(data.get("customAttributes", "{}")),
         )
 
 
+@dataclass
+class FirebaseOAuth:
+    id_token: str
+    access_token: str
+    refresh_token: str = None
```

Comment:
When the `refresh_token` is None?

Reply:
When the OAuth does not explicitly required offline access.